### PR TITLE
boltai 1.15.7 (new cask)

### PR DESCRIPTION
**Pull request by longseespace**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/boltai.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+cask "boltai" do
+  version "1.15.7"
+  sha256 "98f7b5e8a4c9a2d37805e41e42cc0d840b29e2b9294f6656103f4cd06000693d"
+
+  url "https://download.boltai.com/releases/BoltAI-#{version}.dmg"
+  name "BoltAI"
+  desc "Beautiful & powerful AI app for busy professionals"
```
Comment (bevanjkay) [inline Casks/b/boltai.rb:7]:
```suggestion
  desc "ChatGPT client"
```

**Review by bevanjkay — COMMENTED** [review]
@longseespace Thank you for submitting this.

Based on the presence of Sparkle, I assume this has auto-updating functionality?
If so, `auto_updates true` should be added.

**Review by bevanjkay — APPROVED** [review]

**Comment by bevanjkay** [comment]
Thank you @longseespace

**Comment by longseespace** [comment]
Thank you @bevanjkay. This is my first time submitting a Cask. I really like the experience. Amazing work! 🙏

**Comment by longseespace** [comment]
Sorry for a noob question @bevanjkay. Should my users update the app through Homebrew or would they use the built-in auto update mechanism by Sparkle?

I'm not sure how the `auto_updates true` behave.

Thanks.

**Comment by bevanjkay** [comment]
`boltai` will not be upgraded with the standard `brew upgrade` command because it has `auto_updates true`.
If you specify the flag `--greedy` or `--greedy-auto-updates` then it will be upgraded.

**Pull request merged**